src/main.c

```diff
@@ -16,16 +16,18 @@
 #include "common.h"
 #include "protocol/interface.h"
 #include "mixer.h"
-#include "gui/gui.h"
-#include "buttons.h"
+#include "gui/gui.h" //diff1
+#include "buttons.h" //diff2
+                     //diff3
 #include "timer.h"
-#include "autodimmer.h"
-#include "music.h"
+#include "autodimmer.h" //diff4
+#include "music.h"      //diff5
 #include "config/model.h"
+                        //diff6
 #include "config/tx.h"
 #include "config/display.h"
 #include "rtc.h"
-#include "extended_audio.h"
+#include "extended_audio.h" //diff7
```

Comment:
Lint Error:
main.c:26:  Should have a space between // and comment  [whitespace/comments] [4]
main.c:30:  At least two spaces is best between code and comments  [whitespace/comments] [2]
main.c:30:  Should have a space between // and comment  [whitespace/comments] [4]